packages/design-system/src/components/web-components/analyticsTesting.ts

```diff
@@ -0,0 +1,78 @@
+import { UtagContainer } from '../analytics/index';
+import { config } from '../config';
+
+export function testAnalyticsEvents({ tagName, renderComponent, configField, triggerAction }) {
```

Comment:
I don't know if this is actually going to be a good idea, but I was just playing around with different ways of reducing the code duplication when a lot of the same tests need to be performed for all the components that have analytics.

Reply:
I figured you could call it multiple times for different permutations of options (like how the `ds-button` could be a `<button>` or `<a>`)